docs/content.zh/release-notes/flink-1.16.md

```diff
@@ -0,0 +1,278 @@
+
+#### Add advanced function DDL syntax "USING JAR"
+
+##### [FLINK-14055](https://issues.apache.org/jira/browse/FLINK-14055)
+
+In 1.16, we introduced the `CREATE FUNCTION ... USING JAR` syntax to support the dynamic loading of 
+the UDF jar in per job, which is convenient for platform developers to easily achieve UDF management.
+In addition, we also port the `ADD JAR` syntax from SqlClient to `TableEnvironment` side, this allows
+the syntax is more general to Table API users. However, due to inconsistent classloader in StreamExecutionEnvironment
+and TableEnvironment, the `ADD JAR` syntax is not available for Table API program currently, it will
+be resolved by https://issues.apache.org/jira/browse/FLINK-29240
+
+More information about this feature could be found in https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/create/#create-function.
+
+#### Allow passing a ClassLoader to EnvironmentSettings
+
+##### [FLINK-15635](https://issues.apache.org/jira/browse/FLINK-15635)
+
+TableEnvironment introduces a user class loader to have a consistent class loading behavior in table
+programs, SQL Client and SQL Gateway. The user classloader manages all user jars such as jar added
+by `ADD JAR` or `CREATE FUNCTION .. USING JAR ..` statements. User-defined functions/connectors/catalogs
+should replace `Thread.currentThread().getContextClassLoader()` with the user class loader to load classes.
+Otherwise, ClassNotFoundException maybe thrown. The user class loader can be accessed via `FunctionContext#getUserCodeClassLoader`,
+`DynamicTableFactory.Context#getClassLoader` and `CatalogFactory.Context#getClassLoader`.
+
+#### Support Retryable Lookup Join To Solve Delayed Updates Issue In External Systems
+
+##### [FLINK-28779](https://issues.apache.org/jira/browse/FLINK-28779)
+
+Adds retryable lookup join to support both async and sync lookups in order to solve the delayed updates issue in external systems.
+
+#### Make `AsyncDataStream.OutputMode` configurable for table module
+
+##### [FLINK-27622](https://issues.apache.org/jira/browse/FLINK-27622)
+
+It is recommend to set the new option 'table.exec.async-lookup.output-mode' to 'ALLOW_UNORDERED' when
+no stritctly output order is needed, this will yield significant performance gains on append-only streams
+
+#### Harden correctness for non-deterministic updates present in the changelog pipeline
+
+##### [FLINK-27849](https://issues.apache.org/jira/browse/FLINK-27849)
+
+For complex streaming jobs, now it's possible to detect and resolve potential correctness issues before running.
+
+### Connectors
+
+#### Move Elasticsearch connector to external connector repository
+
+##### [FLINK-26884](https://issues.apache.org/jira/browse/FLINK-26884)
+
+The Elasticsearch connector has been copied from the Flink repository to its own individual repository
+at https://github.com/apache/flink-connector-elasticsearch
+
```

Comment:
This needs to be re-phrased. It sounds like a breaking change (i.e., "I _have_ to update my _pom_") when for 1.16 it's mostly a migration path ("I _should_ update my pom _eventually_ to leverage the new connector, because in 1.17 its the only option.").